Completion/Unix/Command/_stow

```diff
@@ -0,0 +1,89 @@
+#compdef stow chkstow
+
+#
+# A zsh completion script for GNU stow (https://www.gnu.org/software/stow/)
+#
+
+__stow_complete_packages() {
```

Comment:
Consider guarding this with `(( $+functions[__stow_complete_packages] )) ||`, so that the function doesn't get re-defined if it already exists. Also, the `complete` in the function name feels a bit redundant

Reply:
done in 3570f596a